src/components/calcite-accordion-item/calcite-accordion-item.scss

```diff
@@ -0,0 +1,168 @@
+// theme variables
+// light theme
+:host {
+  --calcite-accordion-item-background-color: #{$ui-foreground};
+  --calcite-accordion-item-color: #{$ui-text-3};
+  --calcite-accordion-item-color-hover: #{$ui-text-2};
+  --calcite-accordion-item-color-active: #{$ui-text-1};
+  --calcite-accordion-item-border-color: #{$blk-030};
+  --calcite-accordion-item-hover-icon-fill: #{$ui-blue};
+}
+
+// dark theme
+:host([theme="dark"]) {
+  --calcite-accordion-item-background-color: #{$ui-foreground-dark};
+  --calcite-accordion-item-color: #{$ui-text-3-dark};
+  --calcite-accordion-item-color-hover: #{$ui-text-2-dark};
+  --calcite-accordion-item-color-active: #{$ui-text-1-dark};
+  --calcite-accordion-item-border-color: #{$blk-180};
+  --calcite-accordion-item-hover-icon-fill: #{$ui-blue-dark};
+}
+
+// scales
+:host([scale="s"]) {
+  --calcite-accordion-item-spacing-y: #{$baseline/3};
+  --calcite-accordion-item-spacing-x: #{$baseline/3};
+  @include font-size(-3);
+}
+
+:host([scale="m"]) {
+  --calcite-accordion-item-spacing-y: #{$baseline/2};
+  --calcite-accordion-item-spacing-x: #{$baseline/2};
+  @include font-size(-2);
+}
+
+:host([scale="l"]) {
+  --calcite-accordion-item-spacing-y: #{$baseline};
+  --calcite-accordion-item-spacing-x: #{$baseline};
+  @include font-size(-1);
+}
+
+// icon position overrides
+:host([icon-position="start"]) {
+  --calcite-accordion-item-flex-direction: row-reverse;
+  --calcite-accordion-item-icon-rotation: 180deg;
+  --calcite-accordion-item-active-icon-rotation: 270deg;
+  --calcite-accordion-item-icon-rotation-rtl: 0deg;
+  --calcite-accordion-item-active-icon-rotation-rtl: -90deg;
+}
+:host([icon-position="end"]) {
+  --calcite-accordion-item-flex-direction: row;
+  --calcite-accordion-item-icon-rotation: 0deg;
+  --calcite-accordion-item-active-icon-rotation: -90deg;
+  --calcite-accordion-item-icon-rotation-rtl: 180deg;
+  --calcite-accordion-item-active-icon-rotation-rtl: 270deg;
+}
+
+// host styling
+:host {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--calcite-accordion-item-background-color);
+  color: var(--calcite-accordion-item-color);
+  transition: 0.15s all ease-in-out;
+  text-decoration: none;
+  outline: none;
+  position: relative;
+}
+
+// minimal variant styling
+:host([appearance="minimal"]) .accordion-item-content,
```

Comment:
May want to test this in Edge. Last time I checked using multiple selectors with host didn't work in the polyfill.

Reply:
May apply to other places as well.

Reply:
the BEM-y way to do this is to add a `accordion-item-content--minimal` class and key off that

Reply:
This does work in Edge, luckily. Styling based on props *seems* more .... web-component-y, but I can update if needed.